server/app/controllers/TaskSolver.scala

```diff
@@ -71,7 +71,7 @@ class TaskSolver @Inject()(executor: RuntimeSuiteExecutor with DynamicSuiteExecu
     }
   }
 
-  private def getTask(year: Long, taskType: String, timeuuid: String): Future[Option[Task]] = {
+  private def getCachedTask(year: Long, taskType: String, timeuuid: String): Future[Option[Task]] = {
```

Comment:
getCachedTask name does not tell that function also work with database. I intentionally left it as getTask. But let's leave this new name for now.